backend/production-optimiser-api/src/main/java/it/polimi/productionoptimiserapi/entities/AccountRequest.java

```diff
@@ -0,0 +1,23 @@
+package it.polimi.productionoptimiserapi.entities;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+
+@Entity
+@Table(name = "account_requests")
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Builder
+public class AccountRequest extends BaseEntity {
+
+  @Column(unique = true)
+  @NotNull
+  private String email;
+
```

Comment:
add status field with enum APPROVED, DENIED
that way on ticket resolution we don't delete entities from db, just change their status

cant be null